### PR TITLE
Update Latest Releases, add SDK version and Waypoint hash

### PR DESCRIPTION
**Pull request by clay-aptos**

### CODE REVIEW

File: developer-docs-site/docs/releases/devnet-release.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 
 [Aptos Framework Release v1.2.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0)
```
Comment (movekevin) [inline developer-docs-site/docs/releases/devnet-release.md:14]: Is devnet on v1.3 yet? @perryjrandall

File: developer-docs-site/docs/releases/devnet-release.md [hunk 2]
```diff
@@ -13,6 +13,10 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
...
 
+## Aptos SDK
+
+Aptos SDK Release v1.7.2
+
 ## Aptos CLI
 
 |Release | Git Tag | Commit Hash|
```
Comment (gregnazario) [inline developer-docs-site/docs/releases/devnet-release.md:22]: Let's add 1.0.8!  I just released it :)

File: developer-docs-site/docs/releases/devnet-release.md (hunk 1 above)
Comment (perryjrandall) [reply developer-docs-site/docs/releases/devnet-release.md:14]: Yes, it has been for a while

Comment (perryjrandall) [reply developer-docs-site/docs/releases/devnet-release.md:14]: Although we dont technically assign a version, so a hash is more correct here?

**Review by perryjrandall — APPROVED** [review]
Accept with comments

File: developer-docs-site/docs/releases/devnet-release.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:22]: Can I get the values from you or somewhere else?

File: developer-docs-site/docs/releases/devnet-release.md (hunk 1 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:14]: From the internal sheet, using:
`1d0dc2c75da1e397effc8bcfa14114652462654d`

Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md]: Please take another look.

File: developer-docs-site/docs/releases/devnet-release.md (hunk 2 above)
Comment (clay-aptos) [reply developer-docs-site/docs/releases/devnet-release.md:22]: All done.

**Pull request merged**